lib/sen_tweet_web/telemetry.ex

```diff
@@ -32,7 +43,29 @@ defmodule SenTweetWeb.Telemetry do
       summary("vm.memory.total", unit: {:byte, :kilobyte}),
       summary("vm.total_run_queue_lengths.total"),
       summary("vm.total_run_queue_lengths.cpu"),
-      summary("vm.total_run_queue_lengths.io")
+      summary("vm.total_run_queue_lengths.io"),
+
+      # Bitfeels Pipeline Source Metrics
+      counter("bitfeels.pipeline.source.id", tags: [:track]),
+
+      # Bitfeels Pipeline Dispatcher Metrics
+      last_value("bitfeels.pipeline.dispatcher.enqueue.queue_length"),
+      last_value("bitfeels.pipeline.dispatcher.enqueue.pending_demand"),
+
+      last_value("bitfeels.pipeline.dispatcher.dequeue.queue_length"),
+      last_value("bitfeels.pipeline.dispatcher.dequeue.demand"),
+      last_value("bitfeels.pipeline.dispatcher.dequeue.events"),
```

Comment:
I'm still trying to figure out which one of these will be useful. Now that we've surfaced a lot of data we can begin experimenting with the dispatcher and see how it affects the `senpy_response_time`.

The idea is that the dispatcher would react to senpy response time and either send more or less tweets in the hope of keeping the response time flat. This is some of the secret sauce to bitfeels that I haven't figured out yet. Essentially it's a way to provide back pressure for third party APIs which is extremely useful to understand and cool. Right now it just sends 1 tweet 😞

Reply:
Let's keep them all for now while we figure out how to implement the backpressure. Are there modules you're thinking to use, or do we need to brew up something mathy/statistical?

Reply:
I think this is enough data to figure out if changes are making things better or worse. We can customize these metrics if need be.